src/nvim/macros.h

```diff
@@ -35,8 +35,8 @@
 #define LINEEMPTY(p) (*ml_get(p) == NUL)
 
 /// BUFEMPTY() - return TRUE if the current buffer is empty
```

Comment:
The comment is now outdated

Reply:
Agreed, we might even remove the bufempty macro to move it as a function, thus I'll address that later on.

Reply:
Done